src/tlo/methods/breast_cancer.py

```diff
@@ -665,45 +680,53 @@ def apply(self, person_id, squeeze_factor):
 
         df.at[person_id, 'brc_breast_lump_discernible_investigated'] = True
```

Comment:
Shouldn't be this true, only if the biopsy is performed, ie if the consumables are available?

Reply:
Its actually not immediately clear what this property does in the code - it seems to be set in the HSI but then isnt looked at again. Again maybe @andrew-phillips-1 can shed some light?

If this should only be set to true if consumables are available then i can make those changes

Reply:
@EvaJanouskova @joehcollins  I think this property might have been created before consumable availability was a constraint.  If it doesn't affect anything in the model and is just an output, maybe you just add a comment that this being True means that she presented for care with a breast lump but not necessarily that it was investigated with a biopsy.